src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class DeltaOnboardingNewVersionFeaturesViewController: DynamicTableViewController, DeltaOnboardingViewControllerProtocol, ENANavigationControllerWithFooterChild, UIAdaptivePresentationControllerDelegate {
+
+	// MARK: - Attributes
+
+	var finished: (() -> Void)?
+
+	// MARK: - Initializers
+	
+	init(featureVersion: String = "", hasCloseButton: Bool = true) {
+		self.viewModel = DeltaOnboardingNewVersionFeaturesViewModel()
+		self.hasCloseButton = hasCloseButton
+		super.init(nibName: nil, bundle: nil)
+	}
+
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: - View Lifecycle Methods
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		self.navigationController?.presentationController?.delegate = self
+
+		setupView()
+
+		if hasCloseButton {
+			setupRightBarButtonItem()
+		}
+	}
+
+	// MARK: - Overrides
+
+	override var navigationItem: UINavigationItem {
+		navigationFooterItem
+	}
+
+	private lazy var navigationFooterItem: ENANavigationFooterItem = {
+		let item = ENANavigationFooterItem()
+
+		item.primaryButtonTitle = AppStrings.NewVersionFeatures.buttonContinue
+		item.isPrimaryButtonEnabled = true
+		item.isSecondaryButtonHidden = true
+
+		item.title = AppStrings.NewVersionFeatures.title
+
+		return item
+	}()
+	
+	// MARK: - Protocol UIAdaptivePresentationControllerDelegate
+	
+	func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
```

Comment:
This is handled by the NavigationController now

```suggestion
	// MARK: - Protocol DismissHandling
	
	func wasAttemptedToBeDismissed() {
```